examples/oci/deploy-all.sh

```diff
@@ -33,19 +35,35 @@ if [[ "$(basename $PWD)" != "oci" ]]; then
   fi
 fi
 
-# Deploy Jaeger (All-in-One with memory storage)
-echo "🟣 Step 1: Installing Jaeger..."
-helm upgrade --install --force jaeger ./helm-charts/charts/jaeger \
-  --set provisionDataStore.cassandra=false \
-  --set allInOne.enabled=true \
-  --set storage.type=memory \
-  --set-file userconfig="./config.yaml" \
-  -f ./jaeger-values.yaml
+if [[ "$MODE" == "upgrade" ]]; then
```

Comment:
it looks like the two branches of the if/then/else are nearly identical. Can we not implement without branching, e.g. 
```
helm $MODE ...
```
(or have an `if` statement that sets some leading parameters instead of $MODE and then still executes the same commands with these dynamic parameters)